.github/workflows/presubmit.yaml

```diff
@@ -130,6 +130,17 @@ jobs:
       - uses: actions/setup-go@v5
         with:
           go-version-file: 'go.mod'
+      - name: Free Up GitHub Actions Ubuntu Runner Disk Space
+        uses: jlumbroso/free-disk-space@main
```

Comment:
Can we use the market-place free-disk-space in lieu of a personal github action?  My concerns are mostly copy-right and maintenance.
https://github.com/marketplace/actions/free-disk-space-ubuntu

Reply:
![image](https://github.com/user-attachments/assets/6dd10ae7-399c-4079-872c-f3dec821c3ce)
Iam using it.

Reply:
🤦‍♀️  Ok 🤣